docs/templates/tour.md

```diff
@@ -0,0 +1,418 @@
+[authentication credentials](./authentication.md).
+
+This template's agent also runs a startup script, sets environment variables,
+and provides metadata.
+
+The
+[`startup script`](https://registry.terraform.io/providers/coder/coder/latest/docs/resources/agent#startup_script)
+installs [code-server](https://coder.com/docs/code-server), a browser-based
+[VS Code](https://code.visualstudio.com/) app that runs in the workspace. We'll
+give users access to code-server through `coder_app`, later.
+
+The
+[`env`](https://registry.terraform.io/providers/coder/coder/latest/docs/resources/agent#env)
+block sets environments variables for the workspace. We use the data source from
+`coder_workspace` to set the environment variables based on the workspace's
+owner. This way, the owner can make git commits immediately without any manual
+configuration.
+
+Your template can use metadata to show information to the workspace owner. Coder
+displays this metadata in the Coder dashboard. Our template has
+[`metadata`](./agent-metadata.md) blocks for CPU and RAM usage.
+
+## 4. coder_app
+
+A
+[`coder_app`](https://registry.terraform.io/providers/coder/coder/latest/docs/resources/app)
+resource lets a developer use an app from the workspace's Coder dashboard.
+
+![Apps in a Coder workspace](../images/templates/workspace-apps.png)
+
+This is commonly used for [web IDEs](../ides/web-ides.md) such as
+[code-server](https://coder.com/docs/code-server/latest), RStudio, and
+JupyterLab.
+
+To install and code-server in the workspace, remember that we installed it in
+the `startup_script` argument in `coder_agent`. We make it available from a
+workspace with a `coder_app` resource. See [web IDEs](../ides/web-ides.md) for
+more examples.
+
+```hcl
+resource "coder_app" "code-server" {
+  agent_id     = coder_agent.main.id
+  slug         = "code-server"
+  display_name = "code-server"
+  url          = "http://localhost:13337/?folder=/home/${local.username}"
+  icon         = "/icon/code.svg"
+  subdomain    = false
+  share        = "owner"
+
+  healthcheck {
+    url       = "http://localhost:13337/healthz"
+    interval  = 5
+    threshold = 6
+  }
+}
+
+```
+
+You can also use a `coder_app` resource to link to external apps, such as links
+to wikis or cloud consoles.
+
+```hcl
+resource "coder_app" "coder-server-doc" {
+  agent_id     = coder_agent.main.id
+  icon         = "/emojis/1f4dd.png"
+  slug         = "getting-started"
+  url          = "https://coder.com/docs/code-server"
+  external     = true
+}
+
+```
+
+## 5. Persistent and ephemeral resources
+
+Managing the lifecycle of template resources is important. We want to make sure
+that workspaces use computing, storage, and other services efficiently.
+
+We want our workspace's home directory to persist after the workspace is stopped
+so that a developer can continue their work when they start the workspace again.
+
+We do this in 2 parts:
+
+- Our `docker_volume` resource uses the `lifecycle` block with
```

Comment:
Should be:

- Our `docker_volume` resource uses the `lifecycle` block with the